Libraries/LibWeb/HTML/HTMLSelectElement.cpp

```diff
@@ -561,7 +562,7 @@ void HTMLSelectElement::update_inner_text_element()
     // Update inner text element to text content of selected option
     for (auto const& option_element : list_of_options()) {
         if (option_element->selected()) {
-            m_inner_text_element->set_text_content(strip_newlines(option_element->text_content()));
+            m_inner_text_element->set_text_content(strip_newlines(option_element->label()));
```

Comment:
The comment above should be updated or removed.

Reply:
Done.